app/src/main/java/org/jellyfin/mobile/player/PlayerActivity.kt

```diff
@@ -232,9 +261,72 @@ class PlayerActivity : AppCompatActivity() {
                 }
                 return true
             }
+
+            override fun onScroll(e1: MotionEvent?, e2: MotionEvent?, distanceX: Float, distanceY: Float): Boolean {
```

Comment:
Please change `e1` and `e2` to more descriptive names.

Reply:
`e1` and `e2` are the names used in the documentation of `onScroll` method. Are you sure I should change them?

![image](https://user-images.githubusercontent.com/20506602/94242447-a97a4900-ff33-11ea-9ba7-debd72c46de0.png)

Reply:
I think something like `firstMotionEvent` and `secondMotionEvent` is more clear. @Maxr1998 what do you think?

Reply:
Maybe `first` and `current`? Imo no need to state the type in the variable. And even if, then only append `Event`, not the full name.